### ISSUE TITLE
Version 7.3.2 does not create consumers as expected

### ISSUE THREAD

**Issue by arielmoraes** [rebus-org/Rebus.RabbitMq#92]
After upgrading to version 7.3.2 consumers are not created anymore.

To reproduce create a bus with any desired max parallelism.

Expected result when starting with 5 workers:
![image](https://user-images.githubusercontent.com/6741634/149533081-ee2a9933-31fa-4bf2-9e01-f283d99b3df5.png)

Version >= 7.3.2
![image](https://user-images.githubusercontent.com/6741634/149534257-fff3b9e7-216f-48b3-9c9c-dcdc59872c30.png)

**Comment by arielmoraes** [comment]
Yes, actually I think that is the expected behavior now as the lib was changed to use a better push model. 

So only one consumer will spawn and the messages will be distributed amongst the available workers (follows the rules of workers/asynchronous). That's better because it will reduce resource usage.

**Comment by KingOfAraby** [comment]
We had a big performance change after updating from 5.x to 6.x+ and you are telling me this is working as intended?

**Comment by mookid8000** [comment]
@KingOfAraby I can understand that this change in behavior was not what you expected, but please use a more respectful tone, and be curious about why the change was made.

**Comment by KingOfAraby** [comment]
Ok, so I re-iterate my question, how do I use or configure Rebus if I want to use multiple channels on the same connection (which RabbitMq and AMQP 0.9 supports)?
Is there any way for me to setup multiple channels or did you just decide to remove that functionality in your library "to save resources"?

**Comment by mookid8000** [comment]
As you have correctly observed, Rebus' RabbitMQ transport only creates one single consumer.

If you'd like to have more than one RabbitMQ consumer on the same queue within the same process, you can create multiple Rebus instances.

I understand that this might not be ideal for your particular usage scenario. Rebus has to satisfy lots of usage scenarios, and so sometimes there's a conflict, and it becomes less ideal for some of them.
